libs/ui/src/tokens/components/molecules/_phone-input.css

```
@@ -0,0 +1,67 @@
+  --color-phone-input-fg: var(--color-fg-primary);
+  --color-phone-input-bg-hover: oklch(
+    from var(--color-phone-input) calc(l + var(--state-hover)) c h
+  );
+  --color-phone-input-bg-focus: oklch(
+    from var(--color-phone-input) calc(l + var(--state-active)) c h
+  );
+  --color-phone-input-border-hover: var(--color-form-control-border-hover);
+  --color-phone-input-border-focus: var(--color-form-control-border-focus);
+  --color-phone-input-ring: var(--color-form-control-ring);
+  --color-phone-input-input-bg-base: transparent;
+  --color-phone-input-input-bg-base: transparent;
```

Stylelint (17.12.0)

[error] Duplicate custom property "--color-phone-input-input-bg-base" (declaration-block-no-duplicate-custom-properties)

(declaration-block-no-duplicate-custom-properties)